Runtime/Client/LootLockerServerRequest.cs

```diff
@@ -150,8 +150,141 @@ public static T InputUnserializableError<T>() where T : LootLockerResponse, new(
         {
             return Error<T>("Method parameter could not be serialized");
         }
+
+        /// <summary>
+        /// Construct an error response because an rate limit has been hit
+        /// </summary>
+        public static T RateLimitExceeded<T>(string method, int secondsLeftOfRateLimit) where T : LootLockerResponse, new()
+        {
+            return Error<T>(string.Format("You are sending too many requests and are being rate limited for %d seconds. Call was to endpoint %s", secondsLeftOfRateLimit, method));
```

Comment:
Maybe change "Call was to" to "Last call was to", since it's global on all endpoints

Reply:
Yeah, but this isn't about the one that triggered the limiting but the one that wasn't sent due to limiting being in effect and that's not global. But maybe I'm misunderstanding?

Reply:
Good idea, fixed